connection-service/new-admin-vm/Install-ConnectionServer.ps1

```diff
@@ -874,6 +891,23 @@ brokerProductVersion=1.0
 brokerIpaddress=$ipaddressString
 brokerLocale=en_US
 "@
+              
+                $isMfa = $using:enableRadiusMfa
+#                Write-Host "MFA setting is $isMfa"
+#stick in RADIUS MFA related attributes if RADIUS MFA is turned on
+                if($isMfa -eq "True") {
```

Comment:
-eq isn't case sensitive, which I think is a good thing in this case.

Reply:
Yes, I could have used "true" as well but as long as I din't use -ceq I figured we would be ok.